app/Http/Middleware/VerifyCsrfToken.php

```diff
@@ -37,26 +38,25 @@ class VerifyCsrfToken extends Middleware
 	public function handle($request, Closure $next): mixed
 	{
 		if ($request->is('api/*')) {
-			/**
-			 * default value is ''
-			 * we force it in case of the migration has not been done.
-			 */
-			$apiKey = Configs::getValueAsString('api_key');
+			$token = $request->header('Authorization');
+			if ($token === '') {
+				return parent::handle($request, $next);
+			}
 
-			/*
-			 * if apiKey is the empty string we directly return the parent handle.
-			 */
-			if ($apiKey === '') {
+			/** @var User|null $user */
+			$user = User::query()
+				->where('token', '=', $token)
+				->where('token', '!=', '')
+				->first();
```

Comment:
That's why we should have a unique index on the `token` column. (See comment wrt. the migration file.) Also inequality conditions are always less efficient in SQL then equality checks on an indexed column. However, if we store `null` for non-existing tokens, then the inequality condition can be dropped.